README.md

```diff
@@ -91,30 +91,63 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
 
 7.  Finally, setup the environment variables:
 
-    To use the default values for: `$ISISROOT, $ISISDATA, $ISISTESTDATA`, run the ISIS variable initialization script with default arguments.
+    ISIS requires several environment variables to be set in order to run correctly. 
+    The variables include: ISISROOT, ISISDATA, and ISISTESTDATA. 
 
-    To do this, for versions of ISIS 4.2.0 and earlier, use:
+    More information about the ISISDATA environment variable and the ISIS Data Area can be found [here]("#The-ISIS-Data-Area"). 
+    
+    The following steps are only valid for versions of ISIS after 4.2.0.   
+    For older versions of ISIS follow the instructions in [this readme file.](https://github.com/USGS-Astrogeology/ISIS3/blob/adf52de0a04b087411d53f3fe1c9218b06dff92e/README.md)
 
-        python $CONDA_PREFIX/scripts/isis3VarInit.py
+    There are two methods to configure the environment variables for ISIS:
 
-    For versions of ISIS after 4.2.0, use:
+    7.1 Using the provided isisVarInit.py script:
 
-        python $CONDA_PREFIX/scripts/isisVarInit.py
+      To use the default values for: `$ISISROOT, $ISISDATA, $ISISTESTDATA`, run the ISIS variable initialization script with default arguments:
+  
+          python $CONDA_PREFIX/scripts/isisVarInit.py
+  
+      Executing this script with no arguments will result in $ISISDATA=$CONDA\_PREFIX/data, and $ISISTESTDATA=$CONDA\_PREFIX/testdata. The user can specify different directories for both of these optional values:
+  
+          python $CONDA_PREFIX/scripts/isisVarInit.py --data-dir=[path to data directory]  --test-dir=[path to test data   directory]
+  
+      Now every time the isis environment is activated, $ISISROOT, $ISISDATA, and $ISISTESTDATA will be set to the values passed to isisVarInit.py. 
+      This does not happen retroactively, so re-activate the isis environment with one of the following commands:
+  
+          for Anaconda 3.4 and up - conda activate isis
+          prior to Anaconda 3.4 - source activate isis
 
-    Executing this script with no arguments will result in $ISISDATA=$CONDA\_PREFIX/data, and $ISISTESTDATA=$CONDA\_PREFIX/testdata. The user can specify different directories for both of these optional values:
+    7.2 Using `conda env config vars`
 
-    For ISIS 4.2.0 and earlier, use:
+      Conda has a built in method for configuring environment variables that are specific to a conda environment since version 4.8.
+      This version number applies only to the conda package, not to the version of miniconda or anaconda that was installed. 
 
-        python $CONDA_PREFIX/scripts/isis3VarInit.py --data-dir=[path to data directory]  --test-dir=[path to test data directory]
+      To determine if your version of conda is recent enough run:
```

Comment:
I appreciate the verbosity, but do we really need to lay out how to check your conda version and how to update it?  Seems like that reproduces things that the conda docs provide.

Reply:
We've provided some of this to help users get used to using conda to install instead of just rsync. I would be fine with just a link to the conda docs. For this PR, I think everything is totally fine.

Reply:
I agree with leaving it, I suspect there is a non-zero population of isis users who only use conda for ISIS. I could add a link in the docs as well but I also think this is sufficient for now.